### PR TITLE
K8SPXC-343 Readme update

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: charts/pxc-operator/README.md [hunk 1]
```diff
@@ -1,23 +1,49 @@
-# pxc-operator: A chart for installing the Percona XtraDB Cluster Operator
+# pxс-operator: A chart for installing the Percona Kubernetes operator for MySQL XtraDB Cluster
```
Comment (fiowro) [inline charts/pxc-operator/README.md:1]:
```suggestion
# pxс-operator: A chart for installing the Percona Kubernetes Operator, which will deploy and manage Percona XtraDB Cluster - a database clustering solution for MySQL.
```

Comment (cap1984) [reply charts/pxc-operator/README.md]: done

File: charts/pxc-operator/README.md [hunk 2]
```diff
@@ -1,23 +1,49 @@
-# pxc-operator: A chart for installing the Percona XtraDB Cluster Operator
+# pxс-operator: A chart for installing the Percona Kubernetes Operator, which will deploy and manage Percona XtraDB Cluster - a database clustering solution for MySQL.
 
-This Helm Chart will deploy the Percona XtraDB Cluster Operator on Kubernetes.
+This chart implements Percona XtraDB Cluster operator deployment. The Operator itself can be found here:
```
Comment (fiowro) [inline charts/pxc-operator/README.md:3]:
```suggestin
This chart implements the Percona XtraDB Cluster Operator deployment. [Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/index.html) is a database clustering solution for MySQL. The Operator itself can be found here:
```

File: charts/pxc-operator/README.md [hunk 3]
```diff
@@ -1,23 +1,49 @@
-# pxc-operator: A chart for installing the Percona XtraDB Cluster Operator
+# pxс-operator: A chart for installing the Percona Kubernetes Operator, which will deploy and manage Percona XtraDB Cluster - a database clustering solution for MySQL.
```
Comment (fiowro) [inline charts/pxc-operator/README.md:1]:
```suggestion
# pxс-operator: A chart for installing the Percona Kubernetes Operator for Percona XtraDB Cluster.
```

Comment (cap1984) [reply charts/pxc-operator/README.md]: done

File: charts/pxc-operator/README.md [hunk 4]
```diff
@@ -1,23 +1,49 @@
-# pxc-operator: A chart for installing the Percona XtraDB Cluster Operator
+# pxс-operator: A chart for installing Percona Kubernetes Operator for Percona XtraDB Cluster.
```
Comment (fiowro) [inline charts/pxc-operator/README.md:1]:
```suggestion
# pxс-operator: A chart for installing Percona Kubernetes Operator for Percona XtraDB Cluster
```

Comment (fiowro) [reply charts/pxc-operator/README.md:1]: lgtm btw :)

File: charts/pxc-operator/README.md [hunk 5]
```diff
@@ -1,23 +1,49 @@
...
+| `replicaCount`                  | PXC Operator Pod quantity                                                     | `1`                                       |
+| `tolerations`                   | List of node taints to tolerate                                               | `[]`                                      |
+| `resources`                     | Resource requests and limits                                                  | `{}`                                      |
+| `nodeSelector`                  | Labels for Pod assignment                                                     | `{}`                                      |
+| `watchNamespace`                | Set when a different from default namespace is needed to watch                | `""`                                      |
+| `nameOverride`                  | Set if the chart name is not desired to be the operators name                 | `""`                                      |
+| `fullnameOverride`              | By default operator name will contain chart name concatenated with operator name.<br>Set this variable if you want to set the whole bunch of names  | `""`                                      |
```
Comment (delgod) [inline charts/pxc-operator/README.md:40]: please list only options which are configurable via the official operator doc

**Review by delgod — CHANGES_REQUESTED** [review]

File: charts/pxc-operator/README.md [hunk 6]
```diff
@@ -1,23 +1,49 @@
...
+| `image.pullSecrets`             | PXC Operator Pod pull secret                                                  | `[]`                                      |
+| `replicaCount`                  | PXC Operator Pod quantity                                                     | `1`                                       |
+| `tolerations`                   | List of node taints to tolerate                                               | `[]`                                      |
+| `resources`                     | Resource requests and limits                                                  | `{}`                                      |
+| `nodeSelector`                  | Labels for Pod assignment                                                     | `{}`                                      |
+| `watchNamespace`                | Set when a different from default namespace is needed to watch                | `""`                                      |
+| `nameOverride`                  | Set if the chart name is not desired to be the operators name                 | `""`                                      |
```
Comment (fiowro) [inline charts/pxc-operator/README.md:39]:
```suggestion
```
It looks like this option is not officially supported

File: charts/pxc-operator/README.md (hunk 5 above)
Comment (fiowro) [inline charts/pxc-operator/README.md:40]:
```suggestion
```
It looks like this option is not officially supported

File: charts/pxc-operator/README.md [hunk 7]
```diff
@@ -1,23 +1,49 @@
...
+| `fullnameOverride`              | By default operator name will contain chart name concatenated with operator name.<br>Set this variable if you want to set the whole bunch of names  | `""`                                      |
+| `createCRD`                     | Set to false if you don't want the helm chart to automatically create the CRD | `true`                                    |
```
Comment (fiowro) [inline charts/pxc-operator/README.md:41]:
```suggestion
```

File: charts/pxc-operator/README.md [hunk 8]
```diff
@@ -1,23 +1,49 @@
...
+| `image.pullPolicy`              | PXC Operator Container pull policy                                            | `Always`                                  |
+| `image.pullSecrets`             | PXC Operator Pod pull secret                                                  | `[]`                                      |
+| `replicaCount`                  | PXC Operator Pod quantity                                                     | `1`                                       |
+| `tolerations`                   | List of node taints to tolerate                                               | `[]`                                      |
+| `resources`                     | Resource requests and limits                                                  | `{}`                                      |
+| `nodeSelector`                  | Labels for Pod assignment                                                     | `{}`                                      |
+| `watchNamespace`                | Set when a different from default namespace is needed to watch                | `""`                                      |
```
Comment (fiowro) [inline charts/pxc-operator/README.md:38]:
```suggestion
```
Is this option officially supported?

File: charts/pxc-operator/README.md (hunk 5 above)
Comment (cap1984) [reply charts/pxc-operator/README.md]: done

File: charts/pxc-operator/README.md (hunk 6 above)
Comment (cap1984) [reply charts/pxc-operator/README.md]: done

File: charts/pxc-operator/README.md (hunk 5 above)
Comment (cap1984) [reply charts/pxc-operator/README.md]: done

File: charts/pxc-operator/README.md (hunk 7 above)
Comment (cap1984) [reply charts/pxc-operator/README.md]: done

File: charts/pxc-operator/README.md (hunk 8 above)
Comment (cap1984) [reply charts/pxc-operator/README.md]: done

**Review by delgod — APPROVED** [review]

**Pull request merged**